src/devices/Lp55231/Lp55231.cs

```diff
@@ -0,0 +1,168 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Device.I2c;
+using System.IO;
+
+namespace Iot.Device.Lp55231
+{
+    /// <summary>
+    /// Raspberry Pi Motor Hat based on PCA9685 PWM controller
+    /// </summary>
+    public class Lp55231 : IDisposable
+    {
+        /// <summary>
+        /// Default I2C address of Motor Hat
+        /// </summary>
+        public const int I2cAddressBase = 0x32;
+
+        /// <summary>
+        /// Gets the channel of the red element of the specific instance
+        /// </summary>
+        /// <param name="instance">The RGB instance (0-2)</param>
+        /// <returns>The channel number</returns>
+        public static byte RedChannel(byte instance) => (byte)(6 + instance);
```

Comment:
how do you use those 3 functions? Can you add something in the samples and the readme? I can't get my head around how to use those because I most likely don't have the sensor. But an explanation in the readme and a sample would definitely help!

Reply:
I've made these internal. They're basically wrappers around the calculation for the appropriate channel number and only used in the `RGBLed` class. The reason they look so odd is that the red channels are kept separate from the blue and green channels as they latter are driven by the IC's internal charge pump whereas the former is driver directly from Vdd (because blue and green LEDS require higher current). This effectively makes the channel layout GBGBGBRRR.